packages/app/src/pages/popup/index.tsx

```diff
@@ -1,71 +1,62 @@
 import React from 'react';
-import { Box, Text, Button, ArrowIcon, BoxProps } from '@stacks/ui';
+import { Button, Stack, Box, ButtonProps } from '@stacks/ui';
 import { PopupContainer } from '@components/popup/container';
 import { useAnalytics } from '@common/hooks/use-analytics';
 import { ScreenPaths } from '@store/onboarding/types';
 import { useWallet } from '@common/hooks/use-wallet';
 import { AccountInfo } from '@components/popup/account-info';
 import { getAccountDisplayName } from '@stacks/wallet-sdk';
+import { Caption, Title } from '@components/typography';
+import { truncateMiddle } from '@stacks/ui-utils';
+import { IconArrowUp, IconQrcode } from '@tabler/icons';
+import { useAssets } from '@common/hooks/use-assets';
 
-interface TxButtonProps extends BoxProps {
-  variant: 'send' | 'receive';
+interface TxButtonProps extends ButtonProps {
+  kind: 'send' | 'receive';
 }
 
-const TxButton: React.FC<TxButtonProps> = ({ variant, onClick }) => {
+const TxButton: React.FC<TxButtonProps> = ({ kind, onClick, ...rest }) => {
   return (
-    <Button
-      onClick={onClick}
-      mode="primary"
-      mr="base-tight"
-      color="blue"
-      customStyles={{
-        primary: {
-          backgroundColor: '#F2F2FF',
-        },
-        secondary: {},
-        tertiary: {},
-      }}
-    >
-      <ArrowIcon
-        direction={variant === 'send' ? ('up' as any) : ('down' as any)}
-        mr="base-tight"
-        height="12px"
+    <Button onClick={onClick} size="sm" px="base" py="tight" fontSize={2} mode="primary" {...rest}>
+      <Box
+        as={kind === 'send' ? IconArrowUp : IconQrcode}
+        transform={kind === 'send' ? 'unset' : 'scaleY(-1)'}
+        size="14px"
       />
-      {variant === 'send' ? 'Send' : 'Receive'}
+      <Box as="span" ml="tight" fontSize="14px">
+        {kind === 'send' ? 'Send' : 'Receive'}
+      </Box>
     </Button>
   );
 };
 
 export const PopupHome: React.FC = () => {
```

Comment:
I'm always forgetting that this file is `/popup/index` - should we rename the file to `/popup/home`?